src/webapp/pages/settings/SettingsPage.tsx

```diff
@@ -177,19 +192,27 @@ export const SettingsPage: React.FC = () => {
     return (
         <DhisPage>
             {dialogProps && <ConfirmationDialog isOpen={true} maxWidth={"lg"} fullWidth={true} {...dialogProps} />}
+            {showCustomSettings && (
+                <CustomizeSettingsDialog
+                    onSave={saveCustomSettings}
+                    customText={appConfig?.customText ?? {}}
```

Comment:
`useMemo`

Reply:
or simpler, define `const emptyObj = {}` at module level and then `appConfig?.customText ?? emptyObj`